### ISSUE TITLE
Document CI/CD process and flow

### ISSUE THREAD

**Issue by stvnrlly** [GSA/notifications-api#48]

**Comment by rahearn** [comment]
Adding to https://docs.google.com/document/d/1QtgTBVAnT4Dt2IFsniKwDJPaIu_xcb38ii7iCPEKQtQ/edit#heading=h.in37vtihaijv for now, but this might be better served in a repo wiki?

**Comment by rahearn** [comment]
@stvnrlly can you please review the [CI/CD section](https://docs.google.com/document/d/1QtgTBVAnT4Dt2IFsniKwDJPaIu_xcb38ii7iCPEKQtQ/edit#heading=h.vib246fthhtw)?

**Comment by stvnrlly** [comment]
Yes, that's great. I'm working on pulling this into the repo in https://github.com/GSA/notifications-api/pull/96, with a slight preference for avoiding a wiki in favor of a docs folder, if we can.
